lib/mutant/env/bootstrap.rb

```diff
@@ -137,7 +137,7 @@ def expression(scope)
         name = scope_name(scope) or return
 
         unless name.instance_of?(String)
-          warn("#{scope.class}#name from: #{scope.inspect} returned #{name.inspect}. #{SEMANTICS_MESSAGE}")
+          warn("#{scope.class}#name from: #{scope} returned #{name}. #{SEMANTICS_MESSAGE}")
```

Comment:
@backus Because this class was touched so heavily your flag triggered me to go the extra mile and include a nicer way to print out formatted semantics warnings.